_security-plugin/access-control/api.md

```diff
@@ -1179,110 +1179,147 @@ PATCH _plugins/_security/api/securityconfig
 
-Retrieves the current security plugin configuration in JSON format.
+Before you can use the REST API to add, retrieve, update, or delete any distinguished names, you must first add the following line to `opensearch.yml`:
+
+```yml
+plugins.security.nodes_dn_dynamic_config_enabled: true
+```
+
+
+### Get distinguished names
+
+Retrieves all allow listed distinguished names.
 
 #### Request
 
 ```
-GET _plugins/_security/api/securityconfig
+GET _plugins/_security/api/nodesdn
 ```
 
+#### Sample response
 
-### Update configuration
-Introduced 1.0
-{: .label .label-purple }
+```json
+{
+  "cluster1": {
+    "nodes_dn": [
+      "CN=cluster1.example.com"
+    ]
+  }
+}
+```
 
-Creates or updates the existing configuration using the REST API rather than `securityadmin.sh`. This operation can easily break your existing configuration, so we recommend using `securityadmin.sh` instead. See [Access control for the API](#access-control-for-the-api) for how to enable this operation.
+To get the distinguished names from a specific cluster or node, include its name in the request's URL.
 
 #### Request
 
+```
+GET _plugins/_security/api/nodesdn/<cluster-name>
+```
+
+#### Sample response
+
 ```json
-PUT _plugins/_security/api/securityconfig/config
 {
-  "dynamic": {
-    "filtered_alias_mode": "warn",
-    "disable_rest_auth": false,
-    "disable_intertransport_auth": false,
-    "respect_request_indices_options": false,
-    "opensearch-dashboards": {
-      "multitenancy_enabled": true,
-      "server_username": "kibanaserver",
-      "index": ".opensearch-dashboards"
-    },
-    "http": {
-      "anonymous_auth_enabled": false
-    },
-    "authc": {
-      "basic_internal_auth_domain": {
-        "http_enabled": true,
-        "transport_enabled": true,
-        "order": 0,
-        "http_authenticator": {
-          "challenge": true,
-          "type": "basic",
-          "config": {}
-        },
-        "authentication_backend": {
-          "type": "intern",
-          "config": {}
-        },
-        "description": "Authenticate via HTTP Basic against internal users database"
-      }
-    },
-    "auth_failure_listeners": {},
-    "do_not_fail_on_forbidden": false,
-    "multi_rolespan_enabled": true,
-    "hosts_resolver_mode": "ip-only",
-    "do_not_fail_on_forbidden_empty": false
+  "cluster3": {
+    "nodes_dn": [
+      "CN=cluster3.example.com"
+    ]
   }
 }
 ```
 
+
+### Update distinguished names
+
+Adds or updates the specified distinguished names in the cluster's or node's allow list.
+
+#### Request
+
+```json
+PUT _plugins/_security/api/nodesdn/<cluster-name>
+{
+  "nodes_dn": [
+    "CN=cluster3.example.com"
+  ]
+}
+```
+
+#### Sample response
+
+```json
+{
+  "status": "CREATED",
+  "message": "'cluster3' created."
+}
+```
+
+
+### Delete distinguished names
+
+Deletes the cluster's allow listed distinguished names.
```

Comment:
Looks like this deletes the entire list? So maybe "Deletes all allow listed distinguished names."?